Tools/AP_Periph/adsb.cpp

```diff
@@ -92,6 +70,21 @@ void AP_Periph_FW::adsb_update(void)
             }
         }
     }
+
+    const uint32_t now_ms = AP_HAL::millis();
```

Comment:
This needs a comment as to why it's required....
```suggestion
    // sensors such as the uAvionix Ping need to see a heartbeat so they know which system ID they are part of
    const uint32_t now_ms = AP_HAL::millis();
```

Reply:
done